README.md

```diff
@@ -56,9 +53,13 @@ Bundles which provide some form of visual information on an overlay.
 
 ## Utilities
 - [prime-manual-alerts](https://github.com/ProbablePrime/prime-manual-alerts) - Manual emits follow, subscription, host and tip events which prime-alerts can listen to.
+- [prime-socialmedia](https://github.com/ProbablePrime/prime-socialmedia) - A NodeCG social media displayer for popups with information on several social media platforms.
+- [streen](https://github.com/SupportClass/streen) - a centralized interface for interacting with Twitch Chat.
+- [Dashboard](https://github.com/nodecg/dashboard) - A Standalone application to check and use your NodeCG Dashboard.
+- [NodeCG-OBS](https://github.com/nodecg/nodecg-obs) - A OBS-Websocket bower module to connect to multiple OBS instances at the same time.

```

Comment:
Change description to `adds the ability to connect to 1-4 instances of OBS (via obs-websocket) to your NodeCG bundle`

Reply:
Sounds good to me. I am not sure if there are more modules to add, i browsed some common githubs i knew from messing with my layouts, but didn't find a lot more.